node-tests/acceptance/end-to-end-test.js

```diff
@@ -94,10 +94,16 @@ describe('end-to-end', function() {
       let { name: tmpDir } = tmp.dirSync();
       process.chdir(tmpDir);
 
-      return ember('new', 'ee-test-app').then(() => {
+      return ember('new', 'ee-test-app', '--skip-git', '--no-welcome').then(() => {
         process.chdir('ee-test-app');
-
-        return ember('install', `ember-electron@file:${path.join(packageTmpDir, 'ember-electron.tgz')}`);
+        // FIXME: This ember-cli-dependency-checker stuff shouldn't be necessary
+        const packageJsonFileWithPath = path.join(process.cwd(), 'package.json');
+        const packageJson = readJsonSync(packageJsonFileWithPath);
+        //packageJson.devDependencies['ember-cli-dependency-checker'] = '^2.2.1';
+        packageJson.devDependencies['ember-cli-dependency-checker'] = 'quaertym/ember-cli-dependency-checker#^2.2.1';
```

Comment:
If someone could explain it to me, I'd love to know why the commented-out line did not work but this one does. They should be equivalent since that is the main/canonical repository for this addon. It seems like an `npm` bug, but I don't want to investigate it further as it has already been too much of a time sink.